torch/functional.py

```diff
@@ -322,6 +323,28 @@ def isnan(tensor):
     return tensor != tensor
 
 
+def isinf(tensor):
+    r"""Returns a new tensor with boolean elements representing if each element is `+/-INF` or not.
+
+    Arguments:
+        tensor (Tensor): A tensor to check
+
+    Returns:
+        Tensor: A ``torch.ByteTensor`` containing a 1 at each location of `+/-INF` elements.
+
+    Example::
+
+        >>> torch.isinf(torch.Tensor([1, float('inf'), 2]))
+         0
+         1
+         0
+        [torch.ByteTensor of size 3]
+    """
+    if not torch.is_tensor(tensor):
+        raise ValueError("The argument is not a tensor")
+    return (tensor == float('inf')) | (tensor == float('-inf'))
```

Comment:
this is three ops though... maybe this should be done as a custom elementwise op.

Reply:
One alternative would be to do: `tensor.abs() == float('inf')`

Reply:
yeah, we should at least do that. it's saving 1 op :)

Reply:
Done! created new PR #9169

Reply:
Closing this PR